test/framework/src/main/java/org/elasticsearch/AnalysisFactoryTestCase.java

```diff
@@ -228,6 +228,10 @@
         .put("fingerprint",               Void.class)
         // for tee-sinks
         .put("daterecognizer",            Void.class)
+        .put("minhash",                   Void.class)
```

Comment:
I opened https://github.com/elastic/elasticsearch/issues/20149 for minhash, and it turns out ES already exposes the Scandinavian analysis components; it's just that they were not properly marked as multi-term aware ... looks like that's a simple fix.